src/ncp/ncp_base.cpp

```diff
@@ -353,6 +354,7 @@ NcpBase::NcpBase(Instance *aInstance)
     IgnoreError(otSetStateChangedCallback(mInstance, &NcpBase::HandleStateChanged, this));
     otIp6SetReceiveCallback(mInstance, &NcpBase::HandleDatagramFromStack, this);
     otIp6SetReceiveFilterEnabled(mInstance, true);
+    otCliInit(mInstance, &NcpBase::HandleCliOutput_Jump, this);
```

Comment:
```suggestion
    otCliInit(mInstance, &NcpBase::HandleCliOutput, this);
```
No need to name it as something like this. Both methods (static version and non-static version) can be called as `HandleCliOutput`.